### PR TITLE
Meraki scenario guide

### PR DESCRIPTION
**Pull request by kbreit**
##### SUMMARY
A scenario guide for the Meraki modules will provide the following benefits:

- Explain what can be done with the modules
- Provide inspiration for what can be done programmatically with Meraki
- Explain limitations of the API and modules

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
meraki

### CODE REVIEW

File: docs/docsite/rst/scenario_guides/guide_meraki.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+.........................
+
+MR access points are enterprise class, high performance, access points for the enterprise. MR access points have MIMO technology and integrated beamforming built-in for high performance applications. BLE allows for advanced location applications to be developed with no on-premises analytics platforms.
+
+Using the Meraki modules
+------------------------
+
+Meraki modules provide a user-friendly interface to manage your Meraki environment using Ansible. For example, details about SNMP settings for a particular organization can be discovered using the module `meraki_snmp <meraki_snmp_module>`.
+
+.. code_block:: yaml
```
Comment (dmsimard) [inline docs/docsite/rst/scenario_guides/guide_meraki.rst]: Try this maybe ?
```
::
    - name: Query SNMP settings
      meraki_snmp:
          api_key: abc123
          org_name: AcmeCorp
          state: query
          delegate_to: localhost
```
I couldn't find any instance of ``code_block`` by grepping the docs.

Comment (kbreit) [reply docs/docsite/rst/scenario_guides/guide_meraki.rst]: Unscores and dashes are important.

**Comment by dagwieers** [comment]
I don't know how I missed this. Should we merge this and backport this to v2.7 ?

**Comment by kbreit** [comment]
We should. I am going to review it and make sure it's good before we merge and backport though. I should be able to do it in a day or two.

**Comment by kbreit** [comment]
@dagwieers Okay lets merge this and I’ll get a back port into 2.7.

File: docs/docsite/rst/scenario_guides/guide_meraki.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+The "Vault" feature of Ansible allows you to keep sensitive data such as passwords or keys in encrypted files, rather than as plain text in your playbooks or roles. These vault files can then be distributed or placed in source control. See :ref:`playbooks_vault` for more information.
+
+Meraki's API returns a 404 error if the API key is not correct. It does not provide any specific error saying the key is incorrect. If you receive a 404 error, check the API key first.
+
+Returned Data Structures
+........................
+
+Meraki and its related Ansible modules return most information in the form of a list. For example, this is returned information by ``meraki_admin`` querying administrators. It returns a list even though there's only one.
+
+.. code-block:: yaml
```
Comment (dagwieers) [inline docs/docsite/rst/scenario_guides/guide_meraki.rst:105]: I would make this 'json' instead.

**Review by dagwieers — APPROVED** [review]

File: docs/docsite/rst/scenario_guides/guide_meraki.rst (hunk 2 above)
Comment (kbreit) [reply docs/docsite/rst/scenario_guides/guide_meraki.rst:105]: I'm trying to do this in the backport and really struggling. Is json not a valid option or do I need to include quotes?

**Pull request merged**